mesheryctl/pkg/utils/error.go

```diff
@@ -503,3 +504,11 @@ func ErrAppendToSheet(err error, id string) error {
 		[]string{"Error occurred while appending to the spreadsheet", "The credential might be incorrect/expired"},
 		[]string{"Ensure correct append range (A1 notation) is used", "Ensure correct credential is used"})
 }
+
+func ErrClearLine(err error) error {
+	return errors.New(ErrClearLineCode, errors.Alert,
+		[]string{"Failed to clear terminal"},
+		[]string{err.Error()},
+		[]string{"Error occurred while attempting to command-line interface"},
```

Comment:
Thanks for this PR, @Aijeyomah 👍 

Will you please rephrase this error description?